.github/workflows/code_analysis_and_tests.yml

```diff
@@ -32,6 +34,42 @@ jobs:
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
+          upload-chunk-size: 33554432
```

Comment:
Please add a comment here to let the later developer know about this is the default value. 🙏🏻